docs/configuring/README.md

```diff
@@ -202,6 +202,14 @@ And in YAML:
 
 **Note:** A globally-installed instance of ESLint can only use globally-installed ESLint plugins. A locally-installed ESLint can make sure of both locally- and globally- installed ESLint plugins.
 
+## Configuring Processors
```

Comment:
I don't think this is needed any longer. There's not really anything to configure.